gelf/udpwriter.go

```diff
@@ -90,7 +91,7 @@ func (w *GelfWriter) writeChunked(zBytes []byte) (err error) {
 	b := make([]byte, 0, ChunkSize)
 	buf := bytes.NewBuffer(b)
 	nChunksI := numChunks(zBytes)
-	if nChunksI > 128 {
+	if nChunksI > MaxChunksCount {
 		return fmt.Errorf("msg too large, would need %d chunks", nChunksI)
```

Comment:
Should make it clearer

```suggestion
		return fmt.Errorf("msg too large, would need %d chunks which execeeds the limit of %d chunks", nChunksI)
```

Reply:
done